libs/zzz/formula/src/data/wengine/util.ts

```diff
@@ -50,3 +57,15 @@ export function entriesForWengine(key: WengineKey): TagMapNodeEntries {
     ),
   ]
 }
+
+export function cmpSpecialtyCount(key: WengineKey, num: NumNode) {
+  const weCount = own.common.count.sheet(key)
```

Comment:
Kinda confusing function names, maybe cmpSpecialtyAndEquipped or something

Reply:
Guess I should have specified the other function is confusing as well but it's fine